### PR TITLE
Add file upload instructions to file browser UI

### LINKED ISSUE #731: Finish DLP redesign implementation
- [x] dandi/dandiarchive#951
- [x] dandi/dandiarchive#863
- [x] dandi/dandi-api#706  
- [x] #732

### LINKED ISSUE #732: Implement "Add files" DLP design
[Figma](https://www.figma.com/file/rBobiAvSLCsJmyYUYmU1OX/DLP?node-id=504%3A33148)

### PR DESCRIPTION
**Pull request by mvandenburgh**
Closes #732 
Closes #731 
Screenshot from 2022-10-24 14-01-28

### CODE REVIEW

File: web/src/components/FileBrowser/FileUploadInstructions.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    </v-row>
+  </v-sheet>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useDandisetStore } from '@/stores/dandiset';
+
+const store = useDandisetStore();
+const dandisetIdentifier = computed(() => store.dandiset?.dandiset.identifier);
```
Comment (waxlamp) [inline web/src/components/FileBrowser/FileUploadInstructions.vue:61]: There's a small typing problem here. `dandisetIdentifier` has type `string | undefined` but you're assuming it's defined at the point of use in the template.

**Review by waxlamp — APPROVED** [review]
If you can fix the typing issue, that would be good to include here. Otherwise, because this is a PR to non-master branch, I'm fine with you merging it pending a bigger review of that branch when it goes to master.

**Comment by danlamanna** [comment]
To avoid the installation instruction versions becoming stale, should we link to https://www.dandiarchive.org/handbook/10_using_dandi/#dandi-python-client instead?

File: web/src/components/FileBrowser/FileUploadInstructions.vue (hunk 1 above)
Comment (mvandenburgh) [reply web/src/components/FileBrowser/FileUploadInstructions.vue:62]: Fixed https://github.com/dandi/dandi-archive/pull/1342/commits/5c98f19e2097faa605b16ebf80de5a3a21c86989

**Comment by mvandenburgh** [comment]
> To avoid the installation instruction versions becoming stale, should we link to [dandiarchive.org/handbook/10_using_dandi/#dandi-python-client](https://www.dandiarchive.org/handbook/10_using_dandi/#dandi-python-client) instead?

Good idea, I think that makes more sense. I added that link and a link to the file upload instructions.

**Comment by mvandenburgh** [comment]
Updated to only change the text about installing the CLI.

**Comment by waxlamp** [comment]
> @waxlamp and @mvandenburgh - given this is a web app, how about we create the text that it reads in some location separate from the web app, and it can pull the info, which we can update async from development of the app.
> 

If this text were going to be in rapid flux (or if it turns out to be like that) I think we could consider this approach. For a first implementation, I prefer the simplicity of what we have in this PR.

> btw, i love this feature.

🥳

**Review by danlamanna — APPROVED** [review]

**Pull request merged**